proposals/lip-0063.md

```diff
@@ -565,25 +565,25 @@ Once the snapshot block is final, a patched version of Lisk Core v4 will be rele
-
 ### Bootstrap Period
 
-There will be a bootstrap period in which a fixed set of delegates will constitute the set of active delegates. This period will last for one week assuming no missed blocks. This is done by setting the [`initRounds`](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-assets-schema) property in the snapshot block to the corresponding value, which is given by `DPOS_INIT_ROUNDS`. The reason to have this period is that the DPoS module selects delegates only with a valid BLS key outside of the bootstrap period. Thus, without a bootstrap period, the DPoS module could not select any delegates right after the hard fork which would result in a stop of the chain.
+There will be a bootstrap period in which a fixed set of delegates will constitute the set of active delegates. This period will last for one week assuming no missed blocks. This is done by setting the [`initRounds`](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-assets-schema) property in the snapshot block to the corresponding value, which is given by `DPOS_INIT_ROUNDS`. The reason to have this period is to achieve that the DPoS module selects only delegates with a valid BLS key. And this is realized by banning every delegate in the snapshot block. The unbanning is triggered by a _[register keys](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md#register-keys)_ transaction. This way, delegates are forced to submit a _register keys_ transaction, and therefore to register a valid BLS key and a (new) generator key, in order to become an active delegate.
```

Comment:
Should we not be switching terminologies to `validator(s)` instead of `delegate(s)` here, at least for the descriptions for concerned situations occurring post migration?

Reply:
> The reason to have this period is to achieve that the DPoS module selects only delegates with a valid BLS key. And this is realized by banning every delegate in the snapshot block.

The new version of the DPoS module actually does not care about BLS keys at all, as it never checks whether a delegate has a valid BLS key in the selection (to have clean and performant code). Instead, the DPoS module only discards banned and punished delegates. This is also the reason that we ban delegates without BLS keys.

Moreover, maybe I would again add "outside the bootstrap period" to make clear that the DPoS selection algorithm only kicks in then.

Reply:
@sameersubudhi so far, the terminology switch from DPoS to PoS was not made public. Therefore, I would keep the current way to keep the term "delegate" for DPoS validators for now. And then we switch the whole terminology in one go. But I must admit that we, including me, have been quite sloppy with terminology. We mix "validator" and "delegate" quite a bit in this LIP but also the DPoS module LIP.

> The new version of the DPoS module actually does not care about BLS keys at all,

Yes. I thought my statements are not contradicting this fact. But I changed this paragraph and also the next one (this one really needed a fix) to make it hopefully more clear.

Reply:
All good, I resolve this comment.